pkg/routes/logs.go

```diff
@@ -43,7 +44,12 @@ func (l Logs) Install(c *restful.Container) {
 func logFileHandler(req *restful.Request, resp *restful.Response) {
 	logdir := "/var/log"
 	actual := path.Join(logdir, req.PathParameter("logpath"))
-
+	//According to NCC-E003660-B4Y. Although http.Server explicitly sanitizes its input,
+	//before using the "actual" path, it should be ensured that it is within the logdir directory.
+	if !strings.HasPrefix(actual, logdir) {
```

Comment:
this check would miss `/var/logger/...`, if the user-provided bit was `../logger/...`

Reply:
@liggitt Fix has been pushed.

Reply:
Thanks